Description: Do not use logging-clear-handlers to see all logging output and
 disable cover-min-percentage since it require python-nose (>= 1.3): this way
 it will be easier to backport python-urllib3 to Wheezy.
Author: Daniele Tricoli <eriol@mornie.org>
Forwarded: not-needed
Last-Update: 2013-10-16

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,8 +1,8 @@
 [nosetests]
-logging-clear-handlers = true
+# logging-clear-handlers = true
 with-coverage = true
 cover-package = urllib3
-cover-min-percentage = 100
+# cover-min-percentage = 100
 cover-erase = true
 
 [egg_info]
